CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/Constants.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace CallAutomation_AppointmentReminder
+{
+    public static class Constants
```

Comment:
can we use IConfiguration provider and proper DI to inject this using the User Secrets store in .NET?